index.js

```diff
@@ -44,6 +44,7 @@ module.exports = function (config) {
       .option('-o, --open', 'Open the Web GUI after startup')
       .option('-v, --verbose')
       .option('--silent')
+      .option('-o, --disable-javascript', 'If set, disables the execution of JavaScript in Mails')
```

Comment:
The `-o` option is already used by [--open](https://github.com/djfarrelly/MailDev/pull/181/files#diff-168726dbe96b3ce427e7fedce31bb0bcR44)

Reply:
Changed to '-j'